provisioning/cli-dist/resources/github-action-production-role.json

```diff
@@ -41,9 +41,12 @@
                 "StringEquals": {
                   "token.actions.githubusercontent.com:aud": "sts.amazonaws.com"
                 },
-                "StringLike": {
+                "ForAnyValue:StringLike": {
```

Comment:
Tady by mělo stačit pouze `StringLike`, ty hodnoty se vezmou jako `OR`. Ale ten zápis obojího ve `Fm::Sub` se mi taky nezdá.

Reply:
ten zapis som si uz fixol a davam ho do dalsieho PR
![image](https://user-images.githubusercontent.com/19371734/201628887-42af78de-dad4-4cfb-b880-3b7e6206f0d8.png)

Reply:
Ako sa to da zapisat lepsie?
```
                "ForAnyValue:StringLike": {
                  "token.actions.githubusercontent.com:sub": [
                    {"Fn::Sub":  "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/heads/main"},
                    {"Fn::Sub":  "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/tags/v*"}
                  ]
                }
```

Reply:
Myslim ze by to melo vypadat aby to fungovalo:
```
"StringLike": {
        "token.actions.githubusercontent.com:sub": [
            {
                "Fn::Sub": "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/heads/main"
            },
            {
                "Fn::Sub": "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/tags/v*"
            }
        ]
    }
```

Reply:
Takto? @Halama 
```
"StringLike": {
  "token.actions.githubusercontent.com:sub": [
    {"Fn::Sub":  "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/heads/main"},
    {"Fn::Sub":  "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/tags/v*"}
  ]
}
```

Reply:
jj to vypada cajk, melo by byt jen preformatovane to co jsem pastnul vyse?